.github/workflows/push-binary-dispatch.yml

```diff
@@ -26,15 +26,12 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        node-version: [10.x, 11.x, 12.x, 13.x, 14.x, 15.x, 16.x]
+        node-version: [14.x, 16.x]
         os: [windows-latest]
         exclude:
-          # exclude windows node 11 due to neon-sys bug
+          # exclude windows node 14 due to another node-gyp bug
           - os: windows-latest
-            node-version: 11.x
-          # exclude windows node 15 due to another node-gyp bug
-          - os: windows-latest
-            node-version: 15.x
+            node-version: 14.x
           # exclude windows node 16 due to another node-gyp bug
           - os: windows-latest
             node-version: 16.x
```

Comment:
Hi @tplooker 
Per the conversations in slack, only node 14 and 16 are in scope,  but the jest test has problems to run with node 14 and 16 on Windows, so basically this job `build_win_pub_binaries` is gonna be ignored at all, I guess there is no binary for Windows be published.

Reply:
Per conversation with @tplooker , removed this redundant Windows build job.